src/write_gcm.c

```diff
@@ -1923,9 +1924,11 @@ static wg_configbuilder_t *wg_configbuilder_create(int children_num,
   };
   static const char *bool_keys[] = {
       "PrettyPrintJSON",
+      "LogPartialErrors",
```

Comment:
The feature visible to the user is that enabling this flag will get partial errors in the logs. The only functional difference is that the server response will include errors. The server will handle failures the same way regardless of the value of this flag (or of the value of the flag in the RPC request).